virtualnetworks.go

```diff
@@ -48,9 +50,22 @@ func (i *ProjectVirtualNetworkServiceOp) List(projectID string, opts *ListOption
 }
 
 type VirtualNetworkCreateRequest struct {
-	ProjectID   string `json:"project_id"`
+	// ProjectID of the project where the VLAN will be made available.
+	ProjectID string `json:"project_id"`
+
+	// Description is a user supplied description of the VLAN.
 	Description string `json:"description"`
-	Facility    string `json:"facility"`
+
+	// TODO: default Description is null when not specified. Permitting *string here would require changing VirtualNetwork.Description to *string too.
+
+	// Facility in which to create the VLAN. Mutually exclusive with Metro.
+	Facility string `json:"facility,omitempty"`
+
+	// Metro in which to create the VLAN. Mutually exclusive with Facility.
+	Metro string `json:"metro,omitempty"`
+
+	// VLAN ID may be specified when created in a metro. It is remotely incremented otherwise. Must be unique per Metro.
+	VLAN int `json:"vlan,omitempty"`
```

Comment:
This field may be renamed to `VXLAN` before the feature is officially released.

Reply:
Yes - before launch this API field may get renamed from `vlan` to `vxlan`.
If it does, I think we should rename the field `VLAN` to match (`VXLAN`).

Reply:
The reasoning is that `VXLAN` would be more consistent with other areas of the API spec, but `VLAN` is a more accurate name.